data/bootstrap_hwdefinition.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
Okay. I'll do some research on YAML schema descriptors as well to more formally specify the input format.